packages/palette/src/elements/Radio/Radio.tsx

```diff
@@ -19,7 +19,7 @@ import {
 export interface RadioProps extends FlexProps {
   /** Disable interactions */
   disabled?: boolean
-  /** Select the button on render */
+  /** Select the button on render. If the Radio is inside a RadioGroup, use RadioGroup.defaultValue instead. */
```

Comment:
Can this be added to palette docs as an example?

Reply:
There are examples for `RadioGroup`, and most of them show the use of `RadioGroup`.`defaultValue` instead of `Radio`.`selected`.

https://palette.artsy.net/elements/inputs/radiogroup/